doc/cephfs/cephfs-top.rst

```diff
@@ -63,6 +63,60 @@ By default, `cephfs-top` uses `client.fstop` user to connect to a Ceph cluster::
   $ ceph auth get-or-create client.fstop mon 'allow r' mds 'allow r' osd 'allow r' mgr 'allow r'
   $ cephfs-top
 
+Description of Fields
+---------------------
+
+chit : cap hit
+It is the percentage of caps issued over total number of caps
+
+dlease : dentry lease
+It is the percentage of dentry leases handed out over the total dentry lease requests
+
+ofiles : opened files
+It is the number of opened files
```

Comment:
Suggest removing "It is" from all of these.

Reply:
I'll make this edit. 

@neesingh-rh, You don't have to do anything. I'll make sure this gets altered, merged, and backported. Thanks for the contribution!

Reply:
> I'll make this edit.
> 
> @neesingh-rh, You don't have to do anything. I'll make sure this gets altered, merged, and backported. Thanks for the contribution!

I already did push the changes addressing above suggestion. Beyond this, you can take care of this. And pls don't merge until @joscollin or @vshankar approves from @ceph/cephfs . Thanks.